### DISCUSSION TITLE
Issue displaying my custom status to others

### DISCUSSION THREAD

**Discussion by bryn98** [maximmax42/Discord-CustomRP#148] [Q&A]
When playing games on my side it displays the custom activity but when my friend looks at my status it shows the game I'm playing. Is there any way to bypass all games and make CustomRp the highest priority to display? Thank you.
![2022-08-09 00 55 21](https://user-images.githubusercontent.com/104917154/183514672-f3a842bb-0230-4402-aeec-8963bda2a0f6.png)

**Comment by maximmax42** [comment]
I don't think there is a way to set a priority. Some time ago it was that the first launched presence was showing in the profile, then last one, and now I have no idea how Discord picks what to show.

**Reply by bryn98** [reply]
Yes indeed that's kinda annoying that you can't pick what you want to display on Discord Activity 😔 Anyways thanks for clearing things up 😊
